h/static/scripts/vendor/jquery.scrollintoview.js

```diff
@@ -109,29 +109,37 @@
 
 				var animOptions = {};
 
+				var yDir = "none";
```

Comment:
> I am advocating for the complete removal of the bucket bar and have for a long, long time.

Anyway, that's a different discussion. 

This PR is about fixing a few regressions in the current implementation of the current design, which has been broken by our refactorings.

Coming up with a new design is a different issue. Could we keep those discussions separate?